adal/src/main/java/com/microsoft/aad/adal/AcquireTokenWithBrokerRequest.java

```diff
@@ -64,6 +64,10 @@ AuthenticationResult acquireTokenWithBrokerSilent()
                 .isNullOrBlank(mAuthRequest.getUserId())) {
             Logger.v(TAG, "User is specified for background(silent) token request, trying to acquire token silently.");
             authenticationResult = mBrokerProxy.getAuthTokenInBackground(mAuthRequest, brokerEvent);
+            brokerEvent.setServerErrorCode(authenticationResult.getServerErrorCode());
```

Comment:
Add check in case `mBrokerProxy.getAuthTokenInBackground(mAuthRequest, brokerEvent)` returns `null`